experimental/packages/opentelemetry-api-metrics/src/types/Metric.ts

```diff
@@ -112,13 +108,10 @@ export interface Histogram {
   record(value: number, labels?: Labels): void;
 }
 
+// ObservableBase has to be an Object but for now there is no field or method
+// declared.
 /** Base interface for the Observable metrics. */
-export interface ObservableBase {
-  observation: (
-    value: number,
-    labels?: Labels,
-  ) => Observation;
-}
+export type ObservableBase = Record<never, never>;
```

Comment:
`Record<never, never>` only allows the empty object correct?

Reply:
It declares an object type, and it can be extended to have any keys. The reason I use `Record` here is that the linter rules disallow empty interface `interface ObservableBase {}`.